src/Http/Productions/ProductionSuggestionController.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Productions;
+
+use CultuurNet\UDB3\Event\Productions\ProductionEnrichedEventRepository;
+use CultuurNet\UDB3\Event\Productions\SimilaritiesClient;
+use CultuurNet\UDB3\Event\Productions\SuggestionsNotFound;
+use CultuurNet\UDB3\HttpFoundation\Response\JsonLdResponse;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+class ProductionSuggestionController
+{
+
+    /**
+     * @var SimilaritiesClient
+     */
+    private $similaritiesClient;
+    /**
+     * @var ProductionEnrichedEventRepository
+     */
+    private $enrichedEventRepository;
+
+    public function __construct(
+        SimilaritiesClient $similaritiesClient,
+        ProductionEnrichedEventRepository $enrichedEventRepository
```

Comment:
I think it would be better to depend on an interface instead of a concrete implementation?